atomic_reactor/plugins/post_pulp_tag.py

```diff
@@ -0,0 +1,85 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from __future__ import print_function, unicode_literals
+
+from atomic_reactor.plugin import PostBuildPlugin
+from atomic_reactor.plugins.build_orchestrate_build import get_worker_build_info
+from atomic_reactor.constants import PLUGIN_PULP_TAG_KEY
+from atomic_reactor.pulp_util import PulpHandler
+
+
+class PulpTagPlugin(PostBuildPlugin):
+    """
+    Find a platform with a v1-image-id annotation and  and tag that docker_image in pulp as
+    directed by tag_conf. Raise an error if two tags have that annotation.
+
+    """
+
+    key = PLUGIN_PULP_TAG_KEY
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow, pulp_registry_name, pulp_secret_path=None,
+                 username=None, password=None, dockpulp_loglevel=None):
+        """
+        constructor
+
+        :param tasker: DockerTasker instance
+        :param workflow: DockerBuildWorkflow instance
+        :param pulp_registry_name: str, name of pulp registry to use, specified in /etc/
+                                   dockpulp.conf
+        :param pulp_secret_path: path to pulp.cer and pulp.key; $SOURCE_SECRET_PATH otherwise
+        :param username: pulp username, used in preference to certificate and key
+        :param password: pulp password, used in preference to certificate and key
+        """
+        # call parent constructor
+        super(PulpTagPlugin, self).__init__(tasker, workflow)
+        self.pulp_registry_name = pulp_registry_name
+        self.pulp_secret_path = pulp_secret_path
+        self.username = username
+        self.password = password
+
+        self.dockpulp_loglevel = dockpulp_loglevel
+
+    def set_v1_tag(self, v1_image_id):
```

Comment:
Nitpick: this could be named `set_v1_tags`, because it applies the tags from `tag_conf` to the image ID.